### ISSUE TITLE
In the case code's comment include multibyte charset, running code failed

### ISSUE THREAD

**Issue by mogimogitomato** [mactkg/vscode-sonic-pi#5]
In the case code's comment include multibyte charset, 
<a href="https://imgur.com/2TyZ4qO"><img src="https://i.imgur.com/2TyZ4qO.png" title="source: imgur.com" /></a>

when running code I got error message from sonic-pi.

```
Runtime Error: [buffer eval, line 777] - ArgumentError
 invalid byte sequence in UTF-8
/Applications/Sonic Pi.app/app/server/ruby/lib/sonicpi/preparser.rb:25:in `gsub!'
/Applications/Sonic Pi.app/app/server/ruby/lib/sonicpi/preparser.rb:25:in `block in preparse'
/Applications/Sonic Pi.app/app/server/ruby/lib/sonicpi/preparser.rb:22:in `each'
/Applications/Sonic Pi.app/app/server/ruby/lib/sonicpi/preparser.rb:22:in `preparse'
/Applications/Sonic Pi.app/app/server/ruby/lib/sonicpi/runtime.rb:777:in `block in __spider_eval'
```

<a href="https://imgur.com/JQAlKKn"><img src="https://i.imgur.com/JQAlKKn.png" title="source: imgur.com" /></a>

In sonic-pi, when sending code call force_encoding.  
https://github.com/samaaron/sonic-pi/blob/master/app/server/ruby/bin/sonic-pi-server.rb#L333

```ruby
code = args[1].force_encoding("utf-8")
```

So, I add utf-8 encoding in the extension, and send a pull request.  

https://github.com/mactkg/vscode-sonic-pi/pull/4

Best regards.

**Comment by mactkg** [comment]
I merged #4 and released new version v0.0.2 to VS marketplace. Please update the extension and try it on your computer.
https://marketplace.visualstudio.com/items?itemName=mactkg.sonic-pi

**Comment by mogimogitomato** [comment]
Thank you! I update extension and retry, error was not raised and can play music.
